### PR TITLE
build: Add a placeholder rgw role for csv generation

### PR DESCRIPTION
**Pull request by travisn**
**Description of your changes:**
The csv will not properly include the rook-ceph-rgw service account unless there is a role and binding that references the service account. No roles are needed yet for the rgw daemon, so this will add a placeholder only for the purposes of csv generation.

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/development-flow.html#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

**Review by BlaineEXE — APPROVED** [review]

File: deploy/examples/common.yaml [hunk 1]
```diff
@@ -1251,6 +1260,21 @@ subjects:
     name: rook-ceph-purge-osd
     namespace: rook-ceph # namespace:cluster
 ---
+# Allow the osd pods in this namespace to work with configmaps
```
Comment (leseb) [inline deploy/examples/common.yaml:1263]: Wrong copy/paste?

**Review by leseb — CHANGES_REQUESTED** [review]

**Comment by parth-gr** [comment]
Does this new ServiceAccount need to be added in RBAC.yaml?

**Comment by travisn** [comment]
> Does this new ServiceAccount need to be added in RBAC.yaml?

Good question, I thought rbac.yaml was generated so I'm curious why it wasn't added there. @BlaineEXE any ideas? Although I stumbled onto a [curious label](https://github.com/rook/rook/blob/release-1.9/build/rbac/rbac.yaml#L1194) in the rbac.yaml, looks leftover from testing.

**Comment by BlaineEXE** [comment]
> > Does this new ServiceAccount need to be added in RBAC.yaml?
> 
> Good question, I thought rbac.yaml was generated so I'm curious why it wasn't added there. @BlaineEXE any ideas? Although I stumbled onto a [curious label](https://github.com/rook/rook/blob/release-1.9/build/rbac/rbac.yaml#L1194) in the rbac.yaml, looks leftover from testing.

It's possible (but I'm really just guessing) that Helm might do some fancy stuff to remove unused RBAC? Maybe we should look into that.

**Pull request merged**